forum/communications/mongo_models.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from mongoengine import Document, EmbeddedDocument, fields
+
+
+class NamespaceInfo(EmbeddedDocument):
+    user_id = fields.LongField()
+    namespace = fields.StringField(required=True)
```

Comment:
Maybe it will be better to use enum field with "startup" and "investor" options? Now we can put any string in this field